### PR TITLE
openjdk-ea: Update to version 21-2-ea

### PR DESCRIPTION
**Pull request by TamamoNoMae13**
Update `openjdk-ea` to version `21-2-ea`.
Added `openjdk21.json`.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/openjdk-ea.json [hunk 1]
```diff
@@ -1,21 +1,21 @@
 {
-    "description": "Official Early-Access Builds of OpenJDK",
+    "description": "Official production-ready open-source builds of OpenJDK 21",
```
Comment (se35710) [inline bucket/openjdk-ea.json:2]: Please revert back to early access.

Comment (TamamoNoMae13) [reply bucket/openjdk-ea.json]: Thanks for pointing out, I just forgot about that and copy-pasted from `openjdk21.json`

**Comment by TamamoNoMae13** [comment]
Fixed.

**Comment by TamamoNoMae13** [comment]
@se35710 if you are able to, please note any changes I need to make and/or merge it within 3 days (before the next auto update to 20-30 and 21-3)

**Review by se35710 — APPROVED** [review]

**Pull request merged**